stories/carousel/carousel.stories.js

```diff
@@ -0,0 +1,321 @@
+                data-slide="prev">
+            </button>
+            <button 
+                class="fd-button fd-carousel__button sap-icon--slim-arrow-right"
+                data-slide="next">
+            </button>
+        </div>
+    </div>
+</div>
+`;
+
+carouselBottom.storyName = 'Carousel with Page Indicator on Bottom';
+carouselBottom.parameters = {
+    docs: {
+        iframeHeight: 900,
+        storyDescription: `
+`
+    }
+};
+
+export const carouselTop = () => `
+<div style="display: flex; flex-direction: column; align-items: center; background: #CCD1D1;">
+    <h4>Navigation Buttons in Page Indicator</h4>
+    <div class="fd-carousel" data-ride="carousel" style="margin-bottom: 3rem; max-width: 30rem;">
+        <div class="fd-carousel__page-indicator-container">
+            <button 
+                class="fd-button fd-carousel__button sap-icon--slim-arrow-left"
+                data-slide="prev">
+            </button>
+            <ol class="fd-carousel__page-indicators">
+                <li 
+                    data-slide-to="1" 
+                    aria-label="Displaying item 1 of 7" 
+                    class="fd-carousel__page-indicator"></li>
+                <li 
+                    data-slide-to="2" 
+                    aria-label="Displaying item 2 of 7" 
+                    class="fd-carousel__page-indicator"></li>
+                <li 
+                    data-slide-to="3" 
+                    aria-label="Displaying item 3 of 7" 
+                    class="fd-carousel__page-indicator"></li>
+                <li 
+                    data-slide-to="4" 
+                    aria-label="Displaying item 4 of 7" 
+                    class="fd-carousel__page-indicator"></li>
+                <li 
+                    data-slide-to="5" 
+                    aria-label="Displaying item 5 of 7" 
+                    class="fd-carousel__page-indicator fd-carousel__page-indicator--active"></li>
+                <li 
+                    data-slide-to="6" 
+                    aria-label="Displaying item 6 of 7" 
+                    class="fd-carousel__page-indicator"></li>
+                <li 
+                    data-slide-to="7" 
+                    aria-label="Displaying item 7 of 7" 
+                    class="fd-carousel__page-indicator"></li>
+            </ol>
+            <button 
+                class="fd-button fd-carousel__button sap-icon--slim-arrow-right"
+                data-slide="next">
+            </button>
+        </div>
+        <div class="fd-carousel__content" style="min-height: 15.5rem;"></div>
+    </div>
+    <h4>Navigation Buttons in Content</h4>
+    <div class="fd-carousel" data-ride="carousel" style="margin-bottom: 3rem; max-width: 30rem;">
+        <div class="fd-carousel__page-indicator-container">
+            <ol class="fd-carousel__page-indicators">
+                <li 
+                    data-slide-to="1" 
+                    aria-label="Displaying item 1 of 7" 
```

Comment:
I don't think the aria-label here is necessary since it isn't focusable. Instead, we might want to include an aria-live container (just one per carousel) that updates as the pages are flipped to say "Displaying item x of y".